red-hat-storage/cephci
# Adding test to perform cluster operations with snapshot mirroring

Automation of test case - CEPH-83574895 - Configure two-way rbd-mirror on Stand alone CEPH cluster on image (with replicated and ec pools)with snapshot based mirroring and perform cluster operations
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83574896  

Test Steps:
1. Bootstrap two CEPH clusters and setup snapshot based mirroring in between these clusters
2. Create pools and images as specified, enable snapshot based mirroring for all these images
3. Schedule snapshots for each of these images and run IOs on each of the images
4. Perform cluster operations on both the clusters
5. Make sure that snapshot mirroring is not affected by any of the cluster operations

Success Logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-6W8V7I/
